deploy/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -376,23 +376,12 @@ rules:
   - get
   - list
 ---
+# TODO: remove this, once https://github.com/rook/rook/issues/10141
```

Comment:
If this is now just a placeholder clusterrole, it will also need at least one rule or else the downstream build will fail. This is the only clusterrole now for the service account?

Reply:
Yes this is the only clusterRole for `rook-csi-cephfs-plugin-sa` SA

Reply:
added one Rule for now.